Goodboy-Oldboy.js

```diff
@@ -0,0 +1,32 @@
+const dogYearOfBirth = 2018;
+const dogYear = 7;
+let dogYearFuture = 2025;
+let shouldShowResultInDogYears = -1;
+let humanYears =
+  "Your dog will be " +
+  (dogYearFuture - dogYearOfBirth) +
+  " human years old in " +
+  dogYearFuture;
+let dogYears =
+  "Your dog will be " +
+  (dogYearFuture - dogYearOfBirth) * 7 +
+  " dog years old in " +
+  dogYearFuture;
+
+console.log((shouldShowResultInDogYears && dogYears) || humanYears);
```

Comment:
Make one template string, then replace it with the correct outputs.

Try using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator